docs/dictionary/property/minStackFileVersion.lcdoc

```diff
@@ -0,0 +1,36 @@
+Name: minStackFileVersion
+
+Type: property
+
+Syntax: get the minStackFileVersion of <stack>
+
+Summary:
+Get the minimum <stackFileVersion}stack file version> for a stack that
+will allow it to be safely <save|saved> without data loss.
+
+Associations: stack
+
+Introduced: 1.0
+
+OS: mac, windows, linux, ios, android, html5
+
+Platforms: desktop, server, mobile
+
+Example:
+set the stackFileVersion to the minStackFileVersion of this stack
+
+Value (string):
+The <minStackFileVersion> <property> of a <stack> is a LiveCode version
+string. The <minStackFileVersion> <property> is read only and determined
+by the content of the stack file.
+
+Description:
+Use the <minStackFileVersion> <property> to ensure that no data loss
+occurs when <save|saving> a <stack> file. Older <stackFileVersion|stack file versions>
+may not support <save|saving> some newer features. For example, stacks
+with widgets must be saved with at least a stack file version of 8.0.
+
+References: property (glossary), stackFileVersion (property), 
```

Comment:
I think all the references to `stackFileVersion` should be changed to reference the `stack version` glossary entry.